### PR TITLE
hack/errata: Poll for recent errata events

### PR DESCRIPTION
**Pull request by wking**
And push to Slack's Incoming WebHook.

### CODE REVIEW

File: hack/errata.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    with open(path, 'w') as f:
+        json.dump(cache, f, sort_keys=True, indent=2)
+
+
+def run(poll_period=datetime.timedelta(seconds=3600), cache=None, webhook=None, **kwargs):
+    next_time = datetime.datetime.now()
+    while True:
+        _LOGGER.debug('poll for messages')
+        for message in poll(period=2*poll_period, **kwargs):
+            if cache and message['fulladvisory'] in cache or 'bug fix update' not in message['synopsis']:
```
Comment (wking) [inline hack/errata.py:37]: `bug fix update` is a cheap hack.  We should really be comparing `fulladvisory` like `RHBA-2020:0936-04` to Cincinnati URI like:

```console
$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=candidate-4.2' | jq -r '.nodes[] | select(.version == "4.2.26").metadata.url'
https://access.redhat.com/errata/RHBA-2020:0936
```

Although I'm not clear on where the `-04` suffix went.

**Comment by LalatenduMohanty** [comment]
@wking How are you planning to integrate this in our workflow?

**Comment by LalatenduMohanty** [comment]
If this can push a comment to the github PR saying the errata has been shipped will be useful.

**Comment by wking** [comment]
> If this can push a comment to the github PR saying the errata has been shipped will be useful.

It cannot do that yet because:

* No GitHub token yet, although we could create one.
* We'd need logic for the script to identify the appropriate PR, possibly by grepping for `fast` and the version number from the errata synopsis in PR titles?  Possible, but not something I wanted to bite off for v1.

Longer term, yeah, this could comment on PRs, or auto-merge PRs, or file new PRs, or file and merge new PRs, or whatever.  I think we'll start out with "ping us in Slack" and see how that goes.  We can figure out what v2 looks like after we get tired of the manual pings ;).

**Pull request merged**